@xen-orchestra/web/src/components/hosts/panel/card/HostInfoCard.vue

```diff
@@ -0,0 +1,171 @@
+          <VtsCopyButton :value="relativeStartTime" />
+        </template>
+      </VtsCardRowKeyValue>
+      <VtsCardRowKeyValue>
+        <template #key>{{ t('patches') }}</template>
+        <template v-if="noMissingPatches" #value>
+          <UiInfo accent="success">{{ t('patches-up-to-date') }}</UiInfo>
+        </template>
+        <template v-else #value>
+          <UiInfo accent="warning">{{ t('n-missing', nMissingPatches) }}</UiInfo>
+        </template>
+        <template #addons>
+          <VtsCopyButton :value="noMissingPatches ? t('patches-up-to-date') : t('n-missing', nMissingPatches)" />
+        </template>
+      </VtsCardRowKeyValue>
+    </div>
+  </UiCard>
+</template>
+
+<script lang="ts" setup>
+import { useXoHostUtils } from '@/composables/xo-host.composable'
+import { useXoHostCollection } from '@/remote-resources/use-xo-host-collection'
+import { useXoHostMissingPatchesCollection } from '@/remote-resources/use-xo-host-missing-patches-collection'
+import { useXoPoolCollection } from '@/remote-resources/use-xo-pool-collection'
+import VtsCardRowKeyValue from '@core/components/card/VtsCardRowKeyValue.vue'
+import VtsCopyButton from '@core/components/copy-button/VtsCopyButton.vue'
+import VtsEnabledState from '@core/components/enabled-state/VtsEnabledState.vue'
+import VtsIcon from '@core/components/icon/VtsIcon.vue'
+import UiCard from '@core/components/ui/card/UiCard.vue'
+import UiCardTitle from '@core/components/ui/card-title/UiCardTitle.vue'
+import UiInfo from '@core/components/ui/info/UiInfo.vue'
+import UiLink from '@core/components/ui/link/UiLink.vue'
+import UiTag from '@core/components/ui/tag/UiTag.vue'
+import UiTagsList from '@core/components/ui/tag/UiTagsList.vue'
+import { vTooltip } from '@core/directives/tooltip.directive'
+import type { XoHost } from '@vates/types'
+import { computed } from 'vue'
+import { useI18n } from 'vue-i18n'
+
+const { host } = defineProps<{
+  host: XoHost
+}>()
+
+const { t } = useI18n()
+
+const { getHostState, getPowerState, getRelativeStartTime } = useXoHostUtils()
+
+const { useGetPoolById } = useXoPoolCollection()
+
+const { getMasterHostByPoolId, isMasterHost } = useXoHostCollection()
+
+const { hostMissingPatches: missingPatches } = useXoHostMissingPatchesCollection({}, () => host.id)
+
+const pool = useGetPoolById(() => host.$pool)
+
+const isMaster = computed(() => isMasterHost(host.id))
+const masterHost = computed(() => getMasterHostByPoolId(host.$pool))
+const nMissingPatches = computed(() => missingPatches.value.length)
+const noMissingPatches = computed(() => nMissingPatches.value === 0)
+const powerState = computed(() => getPowerState(host.power_state))
+
+const relativeStartTime = host.startTime ? getRelativeStartTime(host.startTime) : undefined
+</script>
+
+<style scoped lang="postcss">
+.card-container {
+  gap: 1.6rem;
+
+  .content {
+    display: flex;
+    flex-direction: column;
+    gap: 0.4rem;
+
+    .power-state,
+    .primary-host {
```

Comment:
```suggestion
    .power-state,
    .pool-name,
    .primary-host {
```
Why are you deleting this class when it's being used? :thinking: